bitcoin-magazine/bitcoin-explained/bitcoin-core-0.21.0-new-release.md

```diff
@@ -0,0 +1,915 @@
+
+Yes, which is ending now.
+
+Sjors Provost: 00:01:53
+
+Probably yes.
+
+Aaron van Wirdum: 00:01:54
+
+That's what I understand.
+The next one, the 22 will actually be Bitcoin Core 22.
+
+Sjors Provost: 00:01:59
+
+That's what it says in the master branch right now but you know until there's a release somebody could come out and you know change it back all right well anyways it's gonna be yeah 22 will be the next version
+
+Aaron van Wirdum: 00:02:10
+
+yeah so this one is still called Bitcoin Core 0.21 and it was released when was it Exactly a week ago when we're recording? No, less than a week ago. Anyways, last week and we're gonna discuss this new release and the new features in this new release.
+
+Sjors Provost: 00:02:27
+
+Yeah, there's some very cool features in there.
+
+Aaron van Wirdum: 00:02:29
+
+It's a big release. There's a lot of new stuff in here.
+
+Sjors Provost: 00:02:32
+
+Yeah, and we're going to refer back to some earlier episodes where we talk about either the specific feature or at least the general area that this feature touches.
+
+Aaron van Wirdum: 00:02:40
+
+Yeah, we already discussed some of it. So what we're going to do for this episode is we're just going to take the release notes and kind of discuss each notable new future point by point for the most interesting ones.
+
+Sjors Provost: 00:02:56
+
+That's right.
+And in the show notes you can find the full release notes which are very long.
+
+Aaron van Wirdum: 00:03:01
+
+Let's start.
+Are you ready?
+
+Sjors Provost: 00:03:03
+
+I'm ready.
+
+Aaron van Wirdum: 00:03:04
+
+Let's go.
+So first points.
+
+## Mempool
+
+Sjors Provost: 00:03:06
+
+Yeah, so the first change, the mempool, right?
+
+It used to be that when you have a Bitcoin Core wallet, you would be broadcasting transactions from your wallet. And then after 15 seconds or so you would be broadcasting transactions from the wallet and you would just keep broadcasting it to your peers nearby.
+
+Aaron van Wirdum: 00:03:26
+
+Isn't it 15 minutes?
+
+Sjors Provost: 00:03:27
+
+Yeah 15 minutes. So it would very frequently broadcast its own transactions because the wallet really wants to make sure that the rest of the world hears about its transactions. The problem with that...
+
+Aaron van Wirdum: 00:03:37
+
+Until it's included in a block, of course.
+
+Sjors Provost: 00:03:40
+
+Exactly. The problem with that is it's not very good for privacy. Because if you can pinpoint which node is constantly repeating a transaction, then that suggests that that's the node that created it.
+
+Aaron van Wirdum: 00:03:51
+
+Yeah, it makes it very obvious which addresses, which inputs originate from which node, which is bad for privacy, because a node can be tied to an IP address and an IP address to an identity and so forth.
+
+Sjors Provost: 00:04:03
+
+Exactly. So the long-term goal here, which has not been achieved in this specific change, but we always do little micro steps.
+
+Aaron van Wirdum: 00:04:11
+
+Uh huh
+
```

Comment:
This kind of lines don't contribute to the transcript's content - you can just delete this and continue with the same speaker. This transcript has a lot back n forth between participants, so you'll notice the presence of numerous filler words or non-essential lines that need removal.